operations/rotate-linux-windows-passwords/README.md

```diff
@@ -0,0 +1,102 @@
+# Simple Password Rotation with HashiCorp Vault
+This guide demonstrates an automated password rotation workflow using Vault and a simple Bash script. These scripts could be run in a cron job or scheduled task to dynamically update local system passwords on a regular basis.
```

Comment:
Is this the same as the other rotate password guide? I remember seeing something very similar, maybe it's the same?

Reply:
I sent a link to the prototype for this, which was stored in my personal github.  This is an attempt to merge it into our hashicorp org.

Reply:
Ahh okay, awesome